docs/source/exposing.md

```diff
@@ -0,0 +1,266 @@
+# Exposing ScyllaCluster
+
+This document explains how ScyllaDB Operator exposes ScyllaClusters in different network setups.
+A ScyllaCluster can be exposed in various network configurations, independently to clients and nodes.
+
+``` note::
+ScyllaClusters can be only exposed when the ScyllaDB version used version is `>=2023.1` ScyllaDB Enterprise or `>=5.2` ScyllaDB Open Source.
+```
+
+## Expose Options
+
+`exposeOptions` as name suggests consists of configuration options for exposing ScyllaCluster's.
+A ScyllaCluster created without any `exposeOptions` is equivalent to the following:
```

Comment:
```suggestion
A ScyllaCluster created without any `exposeOptions` specified takes on the following default values:
```
nit

Reply:
I like submitted one better, it was also suggested to call it this way in other remark.